lang/dart.md

```diff
@@ -0,0 +1,203 @@
+
+**Dart Installation:**
+
+**Install Dart on Windows:**
+In this tutorial, we will provide a step by step process to get Dart installed on Windows.
+Dart SDK comes as a pre-compiled version. So, download and extracting is all you need to do.
+
+**Step 1:** Download Dart SDK
+Go to Dart SDK archive page.
+The URL is [Click Here ](https://dart.dev/tools/sdk/archive)
+Download Dart SDK
+Click on the Dart SDK link. The download will start immediately and a zip file will be downloaded.
+
+**Step 2:** Extract zip file
+Extract the contents of Dart SDK zip file. The contents of the folder would be as shown below.
+
+**Step 3:**
+Add Dart Path to PATH Environment Variable
+As of now, you can run dart command only from bin folder of dart sdk. To run dart command from anywhere in your file system, add dart bin path to PATH environment variable.
+Open Environment Variables. Under System variables, click on Path and click Edit button.
+Edit environment variable window appears. Click on New and paste the dart sdk bin path as shown below.
+
+**Step 4:** Restart Command Prompt
+Close the existing command prompt window and open a new command prompt.
+Just run the command: 
+
+```bash  
+dart --version 
+```
+#### **⚙️ Install the Dart SDK on Linux:**
+
+The steps of Dart installation on Linux is given below.
+Before installing the Dart, if you are Debian/Ubuntu on AMD64(64-bit Intel) in your local machine, you can install the Dart through one of the following options.
+Install using apt-get
+Install a Debian package
+Installation using apt-get
+
+**Step 1:**
+
+ Type the following commands for a one-time setup.
+
+```bash
+sudo apt-get update  
+sudo apt-get install apt-transport-https  
+sudo sh -c 'wget -qO- https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -'  
+sudo sh -c 'wget -qO- https://storage.googleapis.com/download.dartlang.org/linux/debian/dart_stable.list > /etc/apt/sources.list.d/dart_stable.list'  
+```
+
+**Step 2:**
+
+ Type the following command in terminal to install the Dart SDK using apt-get option.
+
+```bash
+sudo apt-get update  
+sudo apt-get install dart
+```
+
+It will successfully download the Dart SDK.
+
+#### **⚙️ Installation a Debian Package:**
+
+We can download Dart SDK as a Debian package in the .deb package format. To make all Dart binaries accessible, we have to change the PATH by typing the following command.
+
+```bash
+export PATH="$PATH:/usr/lib/dart/bin"
+```
+
+To change the PATH for upcoming terminal sessions, use a below command:
+```bash
+echo 'export PATH="$PATH:/usr/lib/dart/bin"' >> ~/.profile
+```
+
+#### **⚙️Install the Dark SDK on Mac**
+
+**Step 1:** We should have Homebrew package manager, but if we don't have it then install the Homebrew and run the following command. It will successfully download the Dart on the Mac.
```

Comment:
Minor spelling typo. Also needs extra space before the code backticks.
```suggestion
**Step 1:** We should have a Homebrew package manager, but if we don't have it then install the Homebrew and run the following command. It will successfully download the Dart on the Mac.
```